src/components/icons.js

```diff
@@ -41,29 +41,29 @@ import MDI_FormatQuote from 'vue-material-design-icons/FormatQuoteClose.vue'
 import MDI_FormatSize from 'vue-material-design-icons/FormatSize.vue'
 import MDI_FormatStrikethrough from 'vue-material-design-icons/FormatStrikethrough.vue'
 import MDI_FormatUnderline from 'vue-material-design-icons/FormatUnderline.vue'
-import MDI_Help from 'vue-material-design-icons/HelpCircle.vue'
+import MDI_Help from 'vue-material-design-icons/HelpCircleOutline.vue'
 import MDI_Image from 'vue-material-design-icons/ImageOutline.vue'
-import MDI_Info from 'vue-material-design-icons/Information.vue'
+import MDI_Info from 'vue-material-design-icons/InformationOutline.vue'
 import MDI_Link from 'vue-material-design-icons/Link.vue'
 import MDI_LinkOff from 'vue-material-design-icons/LinkOff.vue'
-import MDI_LinkVariantPlus from 'vue-material-design-icons/LinkVariantPlus.vue'
+import MDI_LinkVariantPlus from 'vue-material-design-icons/LinkPlus.vue'
 import MDI_Loading from 'vue-material-design-icons/Loading.vue'
-import MDI_Lock from 'vue-material-design-icons/Lock.vue'
+import MDI_Lock from 'vue-material-design-icons/LockOutline.vue'
 import MDI_Paperclip from 'vue-material-design-icons/Paperclip.vue'
-import MDI_Pencil from 'vue-material-design-icons/Pencil.vue'
-import MDI_PencilOff from 'vue-material-design-icons/PencilOff.vue'
+import MDI_Pencil from 'vue-material-design-icons/PencilOutline.vue'
+import MDI_PencilOff from 'vue-material-design-icons/PencilOffOutline.vue'
 import MDI_Plus from 'vue-material-design-icons/Plus.vue'
-import MDI_Shape from 'vue-material-design-icons/Shape.vue'
+import MDI_Shape from 'vue-material-design-icons/ShapeOutline.vue'
 import MDI_Table from 'vue-material-design-icons/Table.vue'
 import MDI_TableSettings from 'vue-material-design-icons/TableCog.vue'
 import MDI_TableAddColumnAfter from 'vue-material-design-icons/TableColumnPlusAfter.vue'
 import MDI_TableAddColumnBefore from 'vue-material-design-icons/TableColumnPlusBefore.vue'
 import MDI_TableAddRowAfter from 'vue-material-design-icons/TableRowPlusAfter.vue'
 import MDI_TableAddRowBefore from 'vue-material-design-icons/TableRowPlusBefore.vue'
-import MDI_TranslateVariant from 'vue-material-design-icons/TranslateVariant.vue'
-import MDI_TrashCan from 'vue-material-design-icons/TrashCan.vue'
+import MDI_TranslateVariant from 'vue-material-design-icons/Translate.vue'
+import MDI_TrashCan from 'vue-material-design-icons/TrashCanOutline.vue'
```

Comment:
This is now imported twice. Once as `MDI_Delete` and once as `MDI_TrashCan`.

Could be imported once and exported twice or the use of one of the two could be changed to the other one.